src/dataframe_builder.py

```diff
@@ -108,15 +112,12 @@ def make_assessments_df(assessments_json, peer_reviews_json, users, rubric, incl
             assessments_df = merged_df.drop(["artifact_id"], axis=1)
     
     else:
-        assessments_df = peer_reviews_df #.drop("asset_id", axis=1)
+        assessments_df = peer_reviews_df
 
     for index, row in assessments_df.iterrows():
         assessments_df.at[index, "Assessor"] = _user_lookup(row["assessor_id"], users)
         assessments_df.at[index, "Assessee"] = _user_lookup(row["user_id"], users)
 
-    #assessments_df = assessments_df.drop(["user_id", "assessor_id"], axis=1)
-    assessments_df = assessments_df.rename(columns={"workflow_state": "State"})
```

Comment:
can you explain why this is dropped?

Reply:
Hi! Sorry for being several months late, I did not get this notification. The commented out part was not used, so I dropped it. The second line I dropped because I combined the command to rename that column to "State" with renaming several other columns (see line 135).